gravitee-apim-console-webui/src/management/application/applications-routing.module.ts

```diff
@@ -50,6 +51,19 @@ const applicationRoutes: Routes = [
       },
     },
   },
+  {
+    path: 'create-old',
+    component: ApplicationCreationAjsComponent,
+    canActivate: [PermissionGuard.checkRouteDataPermissions],
+    data: {
+      permissions: {
+        anyOf: ['environment-application-c'],
+      },
+      docs: {
+        page: 'management-create-application',
+      },
+    },
+  },
```

Comment:
Shouldn't we remove it now and remove the component files too ? 🤷🏼 Wdyt ?

Reply:
I want to keep this in case. or it will be the subject of another PR as said in the description ;)